### PR TITLE
LPS-175275[Site Initializer] Replace logic for Company in Site Initializer

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 1]
```diff
@@ -867,12 +867,13 @@ else if (Objects.equals(type, "widget")) {
 			_replace(
 				json,
 				new String[] {
-					"[$GROUP_FRIENDLY_URL$]", "[$GROUP_ID$]", "[$GROUP_KEY$]"
+					"[$GROUP_FRIENDLY_URL$]", "[$GROUP_ID$]", "[$GROUP_KEY$]",
+					"[$COMPANY_ID$]"
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:871]: Please put it in alphabetic order
Thx

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joseabelenda** [comment]
ci:test:solutions-acceptance

**Review by nNilton — APPROVED** [review]

**Comment by nNilton** [comment]
ci:forward

**Pull request closed without merging**